serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/config/SecurityConfig.java

```diff
@@ -1,14 +1,42 @@
 package no.difi.meldingsutveksling.serviceregistry.config;
 
+import no.difi.meldingsutveksling.serviceregistry.auth.OrgnrUserAuthConverter;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
+import org.springframework.security.oauth2.provider.token.DefaultAccessTokenConverter;
+import org.springframework.security.oauth2.provider.token.RemoteTokenServices;
 
 @ConditionalOnProperty(name = "difi.move.auth.enable", havingValue = "true")
 @Configuration
 @EnableResourceServer
 @EnableGlobalMethodSecurity(prePostEnabled = true)
 public class SecurityConfig {
 
+    @Bean
+    public BeanPostProcessor configureRemoteTokenService() {
+        return new BeanPostProcessor() {
+            @Override
+            public Object postProcessBeforeInitialization(Object o, String string) throws BeansException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'org.springframework.beans.BeansException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)